core/admin.py

```diff
@@ -1,11 +1,43 @@
 from django.contrib import admin
 from django_admin_display import admin_display
 from django.utils.safestring import mark_safe
+from django.db.models import FileField
+import os
 
 from . import models
 from . import tasks
 
 
+def _text_preview(log_file: FileField):
+    """
+    Return the text of a file if it is short or the last portion of it if it is long.
+
+    :params: log_file A FileField to read text from.
+    """
+    # max file size for display, currently 10kb
+    maxlen = 10000
+    prefix_message = 'The output is too large to display in the browser.\n'
+    prefix_message += ('Only the last %s characters are displayed.\n \n' % (maxlen))
```

Comment:
Since prefix_message is only used by one if-else branch, can we move its definition to that location?

Also, you can do this with a single multi-line format string 
```
    prefix_message = f"""The output is too large to display in the browser.
Only the last {maxlen} characters are displayed.

"""
```

Reply:
Yes, I will do that.